securedrop/db.py

```diff
@@ -41,6 +41,8 @@
         config.DATABASE_ENGINE + ":///" +
         config.DATABASE_FILE
     )
+    engine.execute('PRAGMA secure_delete = ON')
```

Comment:
This should be moved to `{journalist,source}_app/__init__.py`. This will behave badly when we complete #2866.

Reply:
And it will probably need to be:

```python
with app.app_context():
    # do stuff
```

Reply:
I don't see how it can be moved to ``{journalist,source}_app/__init__.py`` and it would be great if you could provide a snippet to show me the way. Thanks :-)

Reply:
This *should* work:

```python
from sqlalchemy import text

def create_app(config):
    app = Flask(__name__,
                template_folder=config.JOURNALIST_TEMPLATES_DIR,
                static_folder=path.join(config.SECUREDROP_ROOT, 'static'))

app.config.from_object(config.JournalistInterfaceFlaskConfig)

with app.app_context():
    db_session.execute(text('PRAGMA secure_delete = ON''))
    db_session.exexute(text('PRAGMA auto_vacuum = FULL'))
    db_session.commit()
```

In fact, the `with app.app_context()` might not even be necessary since we have some no wrapper libs. Anyway, calling it directly from the app's init function should work.